### PR TITLE
Release v9.3.0-alpha.1

### PR DESCRIPTION
**Pull request by tobrun**
This PR updates the Mapbox Maps SDK for Android to use a pre-built GL Native binary rather than building from source as a submodule dependency. GL Native is changing some of its development workflows to build some Mapbox platform-specific features internally, similar to Chrome/Chromium development.

#### How to build
Going forward, updates to the build process are as follows:

 - Sign up or login to your Mapbox account.
 - Go to your Mapbox account's access tokens page (https://account.mapbox.com/access-tokens/) and create a secret token with the downloads:read scope.
 - Export this access token as part of your environment variables:
```
# Append to the end of the file
$ echo "export SDK_REGISTRY_TOKEN='{your_token_here}'" >> ~/.bash_profile

# Update shell
$ source ~/.bash_profile
```
  
 - Build the project as you would have before.

#### License update for Mapbox builds of GL Native
New binary releases of GL Native are now licensed under the Mapbox [Terms of Service](https://www.mapbox.com/legal/tos). The Mapbox Maps SDKs repos will continue to be public and licensed under BSD-2. 

If you have any questions about how this change does or doesn’t impact your use of the Maps SDK please don’t hesitate to reach out to [Mapbox Support](https://support.mapbox.com/hc/en-us) to connect with the Mapbox team.

### CODE REVIEW

File: MapboxGLAndroidSDKTestApp/build.gradle [hunk 1]
```diff
@@ -18,8 +17,6 @@ android {
         targetCompatibility JavaVersion.VERSION_1_8
     }
 
-    nativeBuild(["example-custom-layer"])
```
Comment (pengdev) [inline MapboxGLAndroidSDKTestApp/build.gradle:21]: shall we ticket out the tail work of adding back custom layer?

Comment (tobrun) [reply MapboxGLAndroidSDKTestApp/build.gradle:21]: yes: https://github.com/mapbox/mapbox-gl-native-android/issues/452

File: README.md [hunk 2]
```diff
@@ -32,88 +32,48 @@ git submodule update --init --recursive
...
-### Running project
-
-Run the configuration for the `MapboxGLAndroidSDKTestApp` module and select a device or emulator to deploy on. Based on the selected device, the c++ code will be compiled for the related processor architecture. You can see the project compiling in the `View > Tool Windows > Gradle Console`.
-
-More information about building and distributing this project in [DISTRIBUTE.md](DISTRIBUTE.md).
+With the first gradle invocation, gradle will take the value of the `MAPBOX_ACCESS_TOKEN` environment variable and save it to `MapboxGLAndroidSDKTestApp/src/main/res/values/developer-config.xml`. If the environment variable wasn't set, you can edit `developer-config.xml` manually and add your access token to the `mapbox_access_token` resource.
 
 ### Additional resources
 
-#### Using the SDK snapshot
```
Comment (pengdev) [inline README.md:97]: shall we ticket out the tail work to add snapshot back?

Comment (tobrun) [reply README.md:97]: yes: https://github.com/mapbox/mapbox-gl-native-android/issues/453 and https://github.com/mapbox/mapbox-gl-native-android/issues/454

**Comment by pengdev** [comment]
Do we need to clean up the CI configuration since we are not building native code(ccache etc.)

**Comment by tobrun** [comment]
> Do we need to clean up the CI configuration since we are not building native code(ccache etc.) This can be a tail work, and considering we might want to add back custom layer example, we might still need it.

TIcket in https://github.com/mapbox/mapbox-gl-native-android/issues/455 to not loose track

**Pull request merged**